src/spec/spec.js

```diff
@@ -1 +1,30 @@
+import menuGen from '../../helpers/menuGenerator';
 
+let menu = '';
+let menuItem = '';
+
+beforeEach(() => {
+  menu = menuGen();
+  menuItem = [menu[0]];
+});
+
+it('should be an array (e.g. typeof === object)', () => {
```

Comment:
maybe use Array.isArray(menu) and expect it toBe true in this case. since like you said, typeof array will be object.

Reply:
thanks! made the change